Formula/defaultbrowser.rb

```diff
@@ -0,0 +1,30 @@
+class Defaultbrowser < Formula
+  desc "Command-line tool for getting & setting the default browser"
+  homepage "https://github.com/kerma/defaultbrowser"
+  url "https://github.com/kerma/defaultbrowser/archive/v1.0.tar.gz"
+  sha256 "c36cc05e778233f7a3c6109ca78227f82dbdde33a5194dae1457cd83fdf27df2"
+  head "https://github.com/kerma/defaultbrowser.git"
+
+  def install
+    if build.head?
+      system "make"
+      mkdir_p bin.to_s
```

Comment:
`make install` should create the necessary directories if they don't exist. Please report it to upstream.

Reply:
I will report it to the upstream, but until they fix it we can leave this here, right?

Reply:
@fxcoudert In case you want to chime in, I filed the aforementioned bug report [here.](https://github.com/kerma/defaultbrowser/issues/14)

Reply:
https://github.com/kerma/defaultbrowser/pull/15